Automattic/wp-calypso
# Stepper Flow: Export step container

#### Changes proposed in this Pull Request

Created two new components to better structure our new steps:
- StepContainer: borrowed from `StepWrapper`, but different as our flows follow a different logic.
- StepNavigationLink: for "Back", "Skip" and "Next" buttons in the header. 

Unfortunately, there were a couple of calypso dependencies (for example `FormattedHeader`) that I had to solve by passing the component to the container.

#### Testing instructions



* Checkout branch
* `yarn start`
* Go to `http://calypso.localhost:3000/stepper`
* Verify that the steps show correctly



Related to #
